src/Behavioral.Automation/Bindings/NavigationBInding.cs

```diff
@@ -56,7 +56,7 @@ public void NavigateToBaseUrl()
         [When("user opens relative URL \"(.*)\"")]
         public void NavigateToRelativeUrl([NotNull] string url)
         {
-            _driverService.NavigateToRelativeUrl(url);
```

Comment:
Maybe we should remove this method from driver service if it is not used anymore? Why was it changed anyway?

Reply:
The problem is - I would like to start a test from a particular page (not the main page).